### ISSUE TITLE
Links are inaccessible for keyboard users

### ISSUE THREAD

**Issue by BillyLevin** [sveltejs/site-kit#10]
There is a style in here removing the outline on anchor elements when they are focused. This is inaccessible and should be removed / altered in a way that makes focused links stand out.

**Comment by Rich-Harris** [comment]
Fixed via https://github.com/sveltejs/site-kit/pull/15
